File: data/code_pairs/cpp/1575.cpp
#include <iostream>
#include <array>

int main() {
    std::array<int, 12> A, C;

    // Initialize A with values 1 to 12
    for (int i = 0; i < 12; ++i) {
        A[i] = i + 1;
    }

    // Increment each element by 1 and assign to C
    for (int i = 0; i < 12; ++i) {
        C[i] = A[i] + 1;
    }

    // Print elements of C
    for (int i = 0; i < 12; ++i) {
        std::cout << "C[" << i << "] = " << C[i] << std::endl;
    }

    return 0;
}

File: data/code_pairs/fortran/1575.f90
program pd_example
    implicit none
    integer, dimension(1:12) :: A, C
    integer :: i

    ! Initialize array A with values 1 to 12
    A = (/ (I, I = 1, 12) /)
    ! Increment each element by 1 and assign to C
    C = A + 1

    ! Print elements of C
    do i = 1, 12
        print *, "C(", i, ") = ", C(i)
    end do
end program pd_example